pkg/strategy/ewo_dgtrd/strategy.go

```diff
@@ -0,0 +1,409 @@
+package ewo_dgtrd
```

Comment:
go convention: avoid using underscore in the package name

Reply:
done